angular/projects/ng-kinintel/src/lib/components/data-picker/data-picker.component.ts

```diff
@@ -15,178 +15,76 @@ const _ = lodash.default;
 export class DataPickerComponent implements OnInit {
 
     @Input() admin: boolean;
+    @Input() typeMapping: any;
 
     @Output() selected = new EventEmitter();
 
     public Object = Object;
+    public _ = _;
```

Comment:
?

Reply:
This is a strange Angular oddity required when exposing imported members for direct use in view templates (html)